### PR TITLE
Release note for button sizes

### PR DESCRIPTION
**Pull request by chloe0592**
Any help/suggestions with writing the release note for unifying the button sizes, would be much appreciated 🙇‍♀️ 

This is the [closed PR](https://github.com/commercetools/ui-kit/pull/2443) for some more context about the release. 
But I'm also here, for any questions :)

### CODE REVIEW

File: website/src/releases/2023-03/button-sizes.mdx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+date: TBA
+title: Unifying all button sizes
+description: Introducing consistent sizes for all the buttons.
```
Comment (emmenko) [inline website/src/releases/2023-03/button-sizes.mdx:4]:
```suggestion
title: Unifying all UI Kit button sizes
description: Introducing consistent sizes for all the UI Kit buttons.
```

File: website/src/releases/2023-03/button-sizes.mdx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+* default size - 'big'
+* available sizes - 'small', 'medium', 'big'
+* **changes:** none
+
+`SecondaryIconButton`:
+* default size - 'big'
+* available sizes - 'small', 'medium', 'big'
+* **changes:** introduced sizes
+
+As always, if you have questions or feedback, you can open a [GitHub Discussion](https://github.com/commercetools/merchant-center-application-kit/discussions) or a [GitHub Issue](https://github.com/commercetools/merchant-center-application-kit/issues).
```
Comment (emmenko) [inline website/src/releases/2023-03/button-sizes.mdx:34]: Let's point the link to the UI Kit repo

File: website/src/releases/2023-03/button-sizes.mdx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+title: Unifying all button sizes
+description: Introducing consistent sizes for all the buttons.
+type: feature
+topics:
+  - Merchant Center
+  - UI Components
+---
+
+There was no standardized sizes of buttons and that represented difficulty of usage and potential UI issues and bugs. 
+With this release we are introducing fixes for that issue and explaining how these components are being used from now on:
```
Comment (emmenko) [inline website/src/releases/2023-03/button-sizes.mdx:12]:
```suggestion
The UI Kit button components can be rendered in different sizes. However, the size values across the different button components were not consistent.
Starting from version `15.13.3` of UI Kit, the button size props have been reworked as following: 
```

File: website/src/releases/2023-03/button-sizes.mdx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+topics:
+  - Merchant Center
+  - UI Components
+---
+
+There was no standardized sizes of buttons and that represented difficulty of usage and potential UI issues and bugs. 
+With this release we are introducing fixes for that issue and explaining how these components are being used from now on: 
+
+`PrimaryButton`:
+* default size - 'big'
```
Comment (emmenko) [inline website/src/releases/2023-03/button-sizes.mdx:15]: Nit: for values let's use the code syntax (backticks)

```suggestion
* default size - `big`
```

File: website/src/releases/2023-03/button-sizes.mdx [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+* available sizes - 'medium', 'big'
+* **changes:** renaming 'small' size to 'medium'. The 'small' size still works, but it'll be depricated in the future.
```
Comment (emmenko) [inline website/src/releases/2023-03/button-sizes.mdx:17]:
```suggestion
* **changes:** renaming 'small' size to 'medium'. The 'small' size still works, but it'll be deprecated in the future.
```

File: website/src/releases/2023-03/button-sizes.mdx [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+`SecondaryButton`:
+* default size - 'big'
+* available sizes - 'medium', 'big'
+* **changes:** introduced sizes
```
Comment (emmenko) [inline website/src/releases/2023-03/button-sizes.mdx:22]:
```suggestion
* **changes:** new size `medium`
```

File: website/src/releases/2023-03/button-sizes.mdx [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+
+`PrimaryIconButton`:
+* default size - 'big'
+* available sizes - 'small', 'medium', 'big'
+* **changes:** none
+
+`SecondaryIconButton`:
+* default size - 'big'
+* available sizes - 'small', 'medium', 'big'
+* **changes:** introduced sizes
```
Comment (emmenko) [inline website/src/releases/2023-03/button-sizes.mdx:32]:
```suggestion
* **changes:** new size `small` and `medium`
```

**Review by emmenko — COMMENTED** [review]
Thanks for drafting the release notes! 🤗

I left some suggestions but overall looks good to me!

**Comment by chloe0592** [comment]
> Thanks for drafting the release notes! 🤗
> 
> I left some suggestions but overall looks good to me!

@emmenko many many thanks! 🙏 I've updated it here - [595d65a](https://github.com/commercetools/merchant-center-application-kit/pull/3002/commits/595d65ae947ed38531f3b7f3b698b7b96931da2f)

File: website/src/releases/2023-03/button-sizes.mdx [hunk 8]
```diff
@@ -0,0 +1,34 @@
+---
+date: TBA
```
Comment (CarlosCortizasCT) [inline website/src/releases/2023-03/button-sizes.mdx:2]: I believe the failing GitHub job is because of this and it will fail until we set the actual date.

**Review by CarlosCortizasCT — APPROVED** [review]
👏 👏 👏

**Review by kark — APPROVED** [review]
🎉
As a side note, I noticed that we missed some README updates in the ui-kit. I think it would be good to correct them before the release.

**Comment by chloe0592** [comment]
> 🎉 As a side note, I noticed that we missed some README updates in the ui-kit. I think it would be good to correct them before the release.

Ah that's good catch! Thanks! Will do it now 👩‍💻

File: website/src/releases/2023-03/button-sizes.mdx [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+title: Unifying all UI Kit button sizes
+description: Introducing consistent sizes for all the UI Kit buttons.
+type: feature
+topics:
+  - Merchant Center
+  - UI Components
```
Comment (Anshuman71) [inline website/src/releases/2023-03/button-sizes.mdx:8]: We've updated the [list of topics](https://commercetools.atlassian.net/wiki/spaces/DOCS/pages/105818548/Writing+release+notes) for Custom Application release notes.

```suggestion
  - Merchant Center
  - Development
```

**Comment by chloe0592** [comment]
@Anshuman71 please will you let us know here when the release note is ready from your side? Thank you ☺️

**Comment by Anshuman71** [comment]
> @Anshuman71 please will you let us know here when the release note is ready from your side? Thank you ☺️

Hi @chloe0592, I was about to message you, but the commit wasn’t going through due to the conventional commit format. I finally pushed my change and assigned it to you. Please check if the technical accuracy is retained while rewording.

> I couldn't use the table formatting because this repo isn't using the latest docs-kit version.

**Review by Anshuman71 — APPROVED** [review]
Looks good to me.

**Review by emmenko — APPROVED** [review]
Thanks! 💯

File: website/src/releases/2023-03/button-sizes.mdx (hunk 9 above)
Comment (CarlosCortizasCT) [reply website/src/releases/2023-03/button-sizes.mdx]: Looks good to me 👍

File: website/src/releases/2023-03/button-sizes.mdx [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+date: 2023-03-24
+title: Introducing consistent button sizes in the UI Kit
+description: The `size` prop across different Button components now has consistent values.
+type: feature
+topics:
+  - Merchant Center
+  - Development
+---
+
+You can adjust the UI Kit button components' size using the `size` prop—however, the size values across the different button components were inconsistent. For example, the `PrimaryButton` supported the `small` and `big` sizes while the `SecondaryButton` only had the `big` size.
```
[the remark this answers is not in the document]
Comment (chloe0592) [reply website/src/releases/2023-03/button-sizes.mdx:11]: @CarlosCortizasCT yeah I see your point and agree as well. @Anshuman71 is it okay with you if we change it to original? Or maybe rephrase the original a bit better if you think that's needed?

[the remark this answers is not in the document]
Comment (Anshuman71) [reply website/src/releases/2023-03/button-sizes.mdx:11]: Hi @CarlosCortizasCT, @chloe0592, the release note needs to share the previous problem and how we solve it (what's the new behavior).

> The `size` prop on the UI Kit button components lets you set the size. However, the `size` values across different button components were inconsistent. For example, the `PrimaryButton` supported the `small` and `big` sizes while the `SecondaryButton` only had the `big` size.

[the remark this answers is not in the document]
Comment (chloe0592) [reply website/src/releases/2023-03/button-sizes.mdx:11]: Thank you @Anshuman71 I'll update it! 🙏

**Comment by Anshuman71** [comment]
@chloe0592, please double-check the date of release before merging.

**Comment by chloe0592** [comment]
> @chloe0592, please double-check the date of release before merging.

Will do @Anshuman71 , thanks for the heads-up!
